.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: Release Cloudformation
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Conventional Changelog Action
+        id: changelog
+        uses: TriPSs/conventional-changelog-action@v3
+        with:
+          skip-on-empty: 'false'
+          skip-version-file: 'true'
+      - name: Release Cloudformation Templates to S3
+        run: |
+          make release RELEASE_VERSION=${{ steps.changelog.outputs.tag }}
+      - name: Create Github Release
+        uses: actions/create-release@v1
+        if: ${{ steps.changelog.outputs.skipped == 'false' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.changelog.outputs.tag }}
+          release_name: ${{ steps.changelog.outputs.tag }}
+          body: ${{ steps.changelog.outputs.clean_changelog }}
```

Comment:
Worth considering retries and idempotency here. Publishing to S3 could succeed and then release creation could fail. Or vice versa if the order were switched. In theory both of these steps could be "skip on existing" rather than "fail on existing." But it may not be worth the trouble given the added complexity of implementing that.

Reply:
I opted to document manual deploys in cases where the automation fails which I think covers the inevitible case where the automation fails partway through a deploy